x-pack/plugins/observability_solution/investigate_app/common/schema/create.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import * as t from 'io-ts';
+import { investigationResponseSchema } from './investigation';
+
+const createInvestigationParamsSchema = t.type({
+  body: t.type({
+    id: t.string,
+    title: t.string,
+    parameters: t.type({
```

Comment:
nit: should we name it `params`? In rule saved object we have `params`, but I don't know if this is common pattern.